### PR TITLE
[#128397007] Add new instance plans

### PR DESCRIPTION
**Pull request by paroxp**
## What

Story with additional summary: https://www.pivotaltracker.com/story/show/128397007/comments/153339795

We'd like to document, additional plans provided by the CF CLI.
## How to review

See if everything is covered in appropriate manner.
## Who can review

Neither @Jonty nor @paroxp

### CODE REVIEW

**Comment by keymon** [comment]
But I do not see the difference between S, M and L. I assume that is "Small, Medium, Large" but what size exactly? or at least relatively? 

Maybe we shall mention that they match to AWS sizes, or simply copy the spec of AWS.

**Comment by paroxp** [comment]
Added extra section on these.

**Comment by keymon** [comment]
LGTM

File: docs/deploying_services/postgres.md [hunk 1]
```diff
@@ -94,19 +108,27 @@ Use ``cf env APPNAME`` to see the environment variables.
...
 
+## Service update
+
+You may find a desire, to upgrade or downgrade your plan, to fulfill your needs. You can do that, by running the following command:
+
+```
+cf update-service SERVICE_INSTANCE -p NEW_PLAN
+```
 
+By default, this functionality will schedule the change, for the next service maintenance slot, see the next section for details. **This however means, that your service may be locked down for further updates, until the scheuled work will finish.**
```
Comment (keymon) [inline docs/deploying_services/postgres.md]: `s/scheuled/scheduled/`

Comment (paroxp) [reply docs/deploying_services/postgres.md]: Thanks. Done.

**Pull request merged**